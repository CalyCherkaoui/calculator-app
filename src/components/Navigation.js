import React from 'react';
import { Link } from 'react-router-dom';

const Navigation = () => (
  <div className="navbar_wrapper">
    <div className="logo">
      Math
      <span> Magicians</span>
    </div>
    <ul className="nav_list">
      <li className="nav_item"><Link to="/" className="nav_link">Home</Link></li>
      <li className="nav_item"><Link to="/Quote" className="nav_link">Quote</Link></li>
      <li className="nav_item"><Link to="/Calculator" className="nav_link">Calculator</Link></li>
    </ul>
  </div>
);

export default Navigation;
