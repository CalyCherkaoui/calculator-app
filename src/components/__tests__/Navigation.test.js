import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter as Router } from 'react-router-dom';
import Navigation from '../Navigation';

describe(
  'Navigation bar component',
  () => {
    it('Matches Navigation bar snapshot', () => {
      const comp = renderer.create(<Router><Navigation /></Router>);
      expect(comp).toMatchSnapshot();
    });
  },
);
