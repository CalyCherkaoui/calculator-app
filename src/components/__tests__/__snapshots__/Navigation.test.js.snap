// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar component Matches Navigation bar snapshot 1`] = `
<div
  className="navbar_wrapper"
>
  <div
    className="logo"
  >
    Math
    <span>
       Magicians
    </span>
  </div>
  <ul
    className="nav_list"
  >
    <li
      className="nav_item"
    >
      <a
        className="nav_link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav_item"
    >
      <a
        className="nav_link"
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
    <li
      className="nav_item"
    >
      <a
        className="nav_link"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
  </ul>
</div>
`;
